src/browser/Navigators.js

```diff
@@ -177,7 +178,26 @@ export const init =
       ( Card
       , deck
       , flags
-      )
+      );
+    const [deck3, $deck3] =
+      ( Runtime.env.url
+      ? Deck.open
+        ( Card
+        , deck2
+        , { output:
+            { uri: Runtime.env.url
+            , disposition: 'default'
+            , features: ''
+            , ref: null
+            , guestInstanceId: null
+            }
+          , overlay: true
+          , isPinned: true
+          , isInputEmbedded: true
```

Comment:
Last three options `overlay`, `isPinned` and `isInputEmbedded` they have special setting for `about:newtab` tab to make it unclosable & have address bar permanently expanded. For regular tabs those actually would be `false`-es. That being said those field are optional so I suggest to just omit those field in this case and correct defaults will be used.